### PR TITLE
allegro.pl

### PR DESCRIPTION
**Pull request by FadeMind**
Smart mark have many selectors `alt`. 
Order page have some fixes too.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -639,8 +639,12 @@ allegro.pl
 
 INVERT
 i[title="Smart!"]
+[alt*="Smart!"]
+[alt="SMART"]
+[alt="Smart"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:644]:
```suggestion
[alt*="smart" i]
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -639,8 +639,12 @@ allegro.pl
...
 div[class="mpof_ki m389_6m"]
 img[alt="Zakup z pakietem Smart"]
+div[class="sob6jg so1d89 mpof_ki"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:647]:
```suggestion
div.sob6jg.so1d89.mpof_ki
```

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:647]: Will apply tomorrow. Mobile now.

**Comment by FadeMind** [comment]
@Gusted ready to go

**Review by Gusted — APPROVED** [review]
Thanks!

**Pull request merged**